src/app/admin/profile-page/profile-page.component.ts

```diff
@@ -45,7 +63,30 @@ export class ProfilePageComponent implements OnInit {
         this.profile = data[`${this.localId}`];
         this.displayName = this.profile.displayName;
       },err => {
-      this.error = err.message;
-    })
+        this.error = err.message;
+      })
+
+    this.formAddTask = new FormGroup({
+      title: new FormControl(null, [Validators.required]),
+      description: new FormControl(null, [Validators.required])
+    });
+  }
+
+  public addTask(): void {
+    if (this.formAddTask.invalid) {
+      return;
+    }
```

Comment:
It would be better to wrap all this block in if (this.formAddTask.valid) { ... }